files/build_templates/init_cfg.json.j2

```diff
@@ -18,17 +18,25 @@
         }
     },
     "FEATURE": {
-{%- for feature, status in [("sflow", "disabled"), ("telemetry", "enabled")] %}
+{%- for feature, status, autorestart in [("bgp", "enabled", "enabled"),
+                            ("dhcp_relay", "enabled", "enabled"),
+                            {% if include_iccpd == "y" %}("iccpd", "disabled", "enabled"),{% endif %}
+                            ("lldp", "enabled", "enabled"),
+                            {% if include_mgmt_framework == "y" %}("mgmt-framework", "enabled", "enabled"),{% endif %}
+                            {% if include_nat == "y" %}("nat", "disabled", "enabled"),{% endif %}
+                            ("pmon", "enabled", "enabled"),
+                            ("radv", "enabled", "enabled"),
+                            {% if include_restapi == "y" %}("restapi", "disabled", "enabled"),{% endif %}
+                            {% if include_sflow == "y" %}("sflow", "disabled", "enabled"),{% endif %}
+                            ("snmp", "enabled", "enabled"),
+                            ("swss", "enabled", "enabled"),
+                            ("syncd", "enabled", "enabled"),
+                            ("teamd", "enabled", "enabled"),
+                            {% if include_system_telemetry == "y" %}("telemetry", "enabled", "enabled"),{% endif %}
+                            ("database", "enabled", "disabled")] %}
         "{{feature}}": {
-            "status": "{{status}}"
-        }{% if not loop.last %},{% endif -%}
-{% endfor %}
-    },
-    "CONTAINER_FEATURE": {
-{%- for container in ["bgp", "database", "dhcp_relay", "lldp", "nat", "pmon", "radv", "restapi", "sflow",
-                      "snmp", "swss", "syncd", "teamd", "telemetry"] %}
-        "{{container}}": {
-            "auto_restart": "{% if container == "database" %}disabled{% else %}enabled{% endif %}",
+            "status": "{{status}}",
```

Comment:
Should we consider changing "status" to "state"?

Reply:
do you prefer state? I can change to state and seems right call.

Reply:
If you also think it's the right call, then we should change it.